lib/CXGN/BrAPI/v2/Germplasm.pm

```diff
@@ -1082,11 +1152,44 @@ sub _simple_search {
             storageTypes=>\@type_of_germplasm_storage_codes,
             subtaxa=>$_->{subtaxa},
             subtaxaAuthority=>$_->{subtaxaAuthority},
-            synonyms=> \@synonyms,
+            synonyms=> @synonyms ? \@synonyms : [],
             taxonIds=>\@taxons,
         };
     }
     return @data;
 }
 
+sub _process_pedigree_string {
```

Comment:
Brapi pedigree format is mother/father. Any reason to convert to array ?

Reply:
During the storage of a germplasm record, we added to code to validate that the parents referenced in the pedigree string exist in the database.  If they don't then the storage method fails.

Reply:
@MFlores2021 should this logic stay as is?